### PR TITLE
New spec to collect running httpd and corresponding pkg

### PR DESCRIPTION
**Pull request by huali027**
Signed-off-by: Huanhuan Li <huali@redhat.com>

### CODE REVIEW

**Comment by huali027** [comment]
@xiangce, please help to review

File: insights/specs/default.py
Comment (xiangce) [inline insights/specs/default.py:700]: As this change will also impact the existing but deprecated parser&combiner `PackageProvidesJava{All}`, please remove the spec `package_provides_java` of the deprecated parser.

**Review by xiangce — APPROVED** [review]
Approved pending spec approval.

**Pull request merged**
